File: dev/requirements/screen/cases/elapsed-and-planned.2.4.case.swift
#if canImport(SwiftUI) && os(iOS)
import SetlistUI

/// 2.4 — elapsed over planned beside the countdown, in the screen's one m:ss format.
let elapsedAndPlanned_2_4 = RequirementCase(id: "2.4") {
    try MainActor.assumeIsolated {
        try expectScreen(StageScreen(state: StageFixture.state()), slug: "elapsed-and-planned", id: "2.4", region: .clocks)
    }
}
#endif

File: dev/requirements/screen/cases/elapsed-and-planned.2.4.case.dart
import 'package:setlist_ui/setlist_ui.dart';

import '../../shared/requirement_case.dart';
import '../../shared/snapshot.dart';

/// 2.4 — Beside the countdown, elapsed and planned time read the set's position.
final elapsedAndPlanned_2_4 = RequirementCase('2.4', (tester) async {
  await expectRegion(
    tester,
    StageFixture.state(),
    region: StageRegion.clocks,
    slug: 'elapsed-and-planned',
    id: '2.4',
  );
});
